ocs_sample_library_preview/ADHClient.py

```diff
@@ -14,18 +14,18 @@
 from .Types import Types
 from .Users import Users
 
-class OCSClient:
+class ADHClient:
     """
-    A client that handles communication with OCS
+    A client that handles communication with ADH
```

Comment:
I would include a comment that this also can communicate with OCS.  I did wonder if we wanted to keep OCSClient around and just add ADHClient as a seperate file and have them reference stuff.  this would help with backwards compatilibility and not totally ripping the rug out from under people... assuming we are keeping the library name the same

Reply:
I agree, added an ADHClient while keeping and deprecating the OCSClient.